src/python/ImpactX.cpp

```diff
@@ -6,6 +6,7 @@
 #include "pyImpactX.H"
 
 #include <ImpactX.H>
+#include "particles/diagnostics/ReducedBeamCharacteristics.H"
```

Comment:
See code around your patch :)
```suggestion
#include <particles/diagnostics/ReducedBeamCharacteristics.H>
```